src/main/java/gift/service/AuthService.java

```diff
@@ -0,0 +1,85 @@
+package gift.service;
+
+import gift.exception.UserErrorCode;
+import gift.exception.UserException;
+import gift.model.User;
+import gift.model.dto.UserRequestDto;
+import gift.repository.UserDao;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jws;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AuthService {
+
+    private static final String secretKey = "Yn2kjibddFAWtnPJ2AFlL8WXmohJMCvigQggaEypa5E=";
```

Comment:
상수는 대문자로 표기하는게 관례입니다.
```suggestion
    private static final String SECRET_KEY = "Yn2kjibddFAWtnPJ2AFlL8WXmohJMCvigQggaEypa5E=";
```

Reply:
넵!